### DISCUSSION TITLE
forceignore file constantly updated by VsCode extensions

### DISCUSSION THREAD

**Discussion by MartineauThomas** [forcedotcom/salesforcedx-vscode#5801] [Q&A] [answered]
Hello!

We are experiencing strange behavior, the forceignore file is constantly updated by 2 VsCode extensions : 

- Aura components (v61.11)
- Lightning Web Components (v61.11)

When they are enabled both add two lines in the .forceignore 
`**/tsconfig.json`
`**/*.ts`

**- Several of my colleagues are also impacted. Are any of you also impacted?**
**- This behavior does not seem normal to me because forceignore is used to exclude metadata**

We find a solution to avoid that behavior by installing a specific version (v60.15.0) of those extensions. 
<img width="650" alt="Screenshot 2024-08-30 at 12 21 49" src="https://github.com/user-attachments/assets/b2f9bd29-fe4b-4bbc-b0f0-48aa308732ef">
<img width="192" alt="Screenshot 2024-08-30 at 12 22 09" src="https://github.com/user-attachments/assets/63393e7d-246c-4605-92a7-8aa958e20ed3">

**Comment by peternhale** [comment, marked as answer]
@MartineauThomas I have created an issue for this, please follow #5802

**Comment by diyer** [comment]
@MartineauThomas , can you please elaborate how this is impacting any functionality?
